mne/commands/tests/test_commands.py

```diff
@@ -177,4 +178,34 @@ def test_surf2bem():
     check_usage(mne_surf2bem)
 
 
+@slow_test
+@requires_freesurfer
+@sample.requires_sample_data
+def test_watershed_bem():
+    """Test mne watershed bem"""
+    check_usage(mne_watershed_bem)
+    orig_subject_dir = os.getenv('SUBJECTS_DIR', None)
+    orig_subject = os.getenv('SUBJECT', None)
+    # Copy necessary files to tempdir
+    tempdir = _TempDir()
+    subjects_dir = op.join(sample.data_path(download=False), 'subjects')
+    mridata_path = op.join(subjects_dir, 'sample', 'mri')
+    mridata_path_new = op.join(tempdir, 'sample', 'mri')
+    os.mkdir(op.join(tempdir, 'sample'))
+    os.mkdir(mridata_path_new)
+    if op.exists(op.join(mridata_path, 'T1')):
+        shutil.copytree(op.join(mridata_path, 'T1'), op.join(mridata_path_new,
+                        'T1'))
+    if op.exists(op.join(mridata_path, 'T1.mgz')):
+        shutil.copyfile(op.join(mridata_path, 'T1.mgz'),
+                        op.join(mridata_path_new, 'T1.mgz'))
+
+    with ArgvSetter(('-d', tempdir, '-s', 'sample', '-o'),
+                    disable_stdout=False, disable_stderr=False):
+        mne_watershed_bem.run()
+
+    os.environ['SUBJECTS_DIR'] = orig_subject_dir
+    os.environ['SUBJECT'] = orig_subject
```

Comment:
Can you see if it's possible to run this on the T1.mgz that is in the `testing` dataset? It is downsampled so it might fail, but if it passes it would be great because it might run faster. How long does this test take on your machine?